### DISCUSSION TITLE
Quarkus Native with Podman for Windows

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#34536] [Quarkus Blog/Website]
# Quarkus Native with Podman for Windows

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/podman-for-windows/

**Comment by bestinlalu** [comment]
Hello, 
Thank you for writing this blog. This is really helpful.
Can you please share the repo of what you have explained here?
I am getting error while trying to test the ITs in podman desktop with the following error:
Errors: 
[ERROR]   GreetingResourceIT.testHelloEndpoint ┬╗ Runtime java.io.IOException: Cannot run program "C:\Users\xxxx\Workplace\Documents\Workspace\code-with-quarkus\target\code-with-quarkus-1.0.0-SNAPSHOT-runner"
: CreateProcess error=193, %1 is not a valid Win32 application

Comparing with your full output you have shared, I see that I am missing lines from
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] Starting (local) container image build for native binary using docker.

Can you please help on this? Thank you
